src/NServiceBus.Core/Licensing/LicenseExpiredForm.cs

```diff
@@ -23,21 +24,27 @@ protected override void OnLoad(EventArgs e)
         {
             base.OnLoad(e);
 
-            warningText.Text = "The trial period is now over";
+            var version = Assembly.GetExecutingAssembly().GetName().Version;
```

Comment:
This will get the assembly version, which is always `6.0.0` , shouldn't this be the fileversion?